Tests/Logging/FileLogHandlerTests.cs

```diff
@@ -64,5 +66,52 @@ public void UsesGlobalFilePath()
 
             File.Delete(Log.FilePath);
         }
+
+        [TestCase(10)]
+        [TestCase(100)]
+        [Ignore("Long logging speed performance test.")]
+        public void TestLoggingSpeeds(int x)
+        {
+            var start = DateTime.Now;
+            const string file = "log2.txt";
+            List<Thread> threads = new List<Thread>();
+
+            //Delete the file if it exists.
+            File.Delete(file);
+
+            using (var log = new FileLogHandler(file))
+            {
+                //Spin off x threads that will use the log handler
+                for (int i = 0; i < x; i++)
+                {
+                    var threadNumber = i;
+                    Thread thread = new Thread(() => {
+                        for (int j = 0; j < 1000; j++)
+                        {
+                            var debugMessage = $"debug message {j} for thread {threadNumber}";
+                            log.Debug(debugMessage);
+                        }
+                    })
+                    {
+                        IsBackground = true,
+                        Name = $"LogTestThread{i}"
+                    };
+
+                    thread.Start();
+                    threads.Add(thread);
+                }
+
+                //Wait for all threads to complete
+                foreach (var thread in threads)
+                {
+                    thread.Join();
+                }
+            }
+
+            var end = DateTime.Now;
```

Comment:
Should always use `UtcNow` -- it sucks when you run a test and you just happen to cross over daylight savings time :)